bonfire/processor.py

```diff
@@ -30,23 +30,82 @@ def _process_template(*args, **kwargs):
     return processed_template
 
 
-def _remove_resource_config(items):
-    for i in items:
-        if i["kind"] != "ClowdApp":
+def _is_trusted_config(value, regex, component_params, path):
+    """
+    Check for presence of a trusted param being used for the value.
+
+    A trusted parameter is defined as:
+    1. matching the expected regex pattern
+    2. the parameter value is set in the component's deploy config
+
+    Returns True if we determine this is a trusted value
+    """
+    in_params = False
+    match = re.match(regex, value)
```

Comment:
Do you need to maybe check that value and regex aren't `None`? I'm not sure. Just wondering in the absence of a type system we could end up with unexpected results of the params aren't what we expect.

Reply:
Good point here. `regex` should never be `None` because we pull it out of the config, but I'll add a type check. `value` could possibly be null/empty so I'll check for that.